testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -142,46 +138,45 @@ def test_old_SegmentGroup_init_works():
 
 class TestAtomGroupToTopology(object):
     """Test the conversion of AtomGroup to TopologyObjects"""
-    @dec.skipif(parser_not_found('DCD'),
-                'DCD parser not available. Are you using python 3?')
-    def setUp(self):
-        self.u = mda.Universe(PSF, DCD)
-
-    def tearDown(self):
-        del self.u
+    @pytest.mark.skipif(parser_not_found('DCD'),'DCD parser not available. Are you using python 3?')
```

Comment:
On my local system, I only have the minimal dependencies, all the test methods are skipped.